lib/helpers/encrypt.js

```diff
@@ -0,0 +1,13 @@
+import bcrypt from 'bcrypt'
+
+class EncryptData {
```

Comment:
I believe since this is oop, the encryptData will always expect some data to either hash or compare... I mean, this could always define a constructor that sets property data in the case unhashed password that should be hashed within the flow or compared to... Would it make sense to have unhashedPassword as a property @Rodgers-M

Reply:
It would make sense having a constructor if we intended to instantiate the class, which we don't anyway coz all of its methods are static.

> Would it make sense to have unhashedPassword as a property

I think it won't because the value is not used anywhere else in the class outside the method and so having it as an argument to the method makes more sense.